src/exam/ExamWrapper.jsx

```diff
@@ -36,12 +36,16 @@ const ExamWrapper = ({ children, ...props }) => {
 
 ExamWrapper.propTypes = {
   sequence: PropTypes.shape({
-    id: PropTypes.string.isRequired,
+    id: PropTypes.string,
```

Comment:
this can be marked as required now with no default props right?

Reply:
yeah, reverted it in a229462